### ISSUE TITLE
Launching refine from eclipse throws an error, while everything work is started form terminal

### ISSUE THREAD

**Issue by EvgenyKarataev** [ColFusion/ColfusionOpenRefine#2]
After stating refine in eclipse, the console log include following exception:

```
java.lang.NoClassDefFoundError: org/apache/juli/logging/LogFactory
    at org.apache.jasper.servlet.JspServlet.<init>(JspServlet.java:68)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
    at java.lang.Class.newInstance(Class.java:442)
    at org.mortbay.jetty.servlet.Holder.newInstance(Holder.java:153)
    at org.mortbay.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:428)
    at org.mortbay.jetty.servlet.ServletHolder.doStart(ServletHolder.java:263)
    at org.mortbay.component.AbstractLifeCycle.start(AbstractLifeCycle.java:50)
    at org.mortbay.jetty.servlet.ServletHandler.initialize(ServletHandler.java:685)
    at org.mortbay.jetty.servlet.Context.startContext(Context.java:140)
    at org.mortbay.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1250)
    at org.mortbay.jetty.handler.ContextHandler.doStart(ContextHandler.java:517)
    at org.mortbay.jetty.webapp.WebAppContext.doStart(WebAppContext.java:467)
    at org.mortbay.component.AbstractLifeCycle.start(AbstractLifeCycle.java:50)
    at org.mortbay.jetty.handler.HandlerWrapper.doStart(HandlerWrapper.java:130)
    at org.mortbay.jetty.Server.doStart(Server.java:224)
    at org.mortbay.component.AbstractLifeCycle.start(AbstractLifeCycle.java:50)
    at com.google.refine.RefineServer.init(Refine.java:203)
    at com.google.refine.Refine.init(Refine.java:116)
    at com.google.refine.Refine.main(Refine.java:110)
Caused by: java.lang.ClassNotFoundException: org.apache.juli.logging.LogFactory
    at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 22 more
```

However, starting from terminal is fine.

**Comment by EvgenyKarataev** [comment]
After #5, now when try to start refine by right clicking on Refine.java and selecting Run as -> Java appliction the following error message is reported in the console window:

`11:06:47.209 [            refine_server] Warning: Failed to find web application at /Users/Evgeny/Documents/Colfusion/ColfusionOpenRefine/server/webapp' (18ms)`

**Comment by EvgenyKarataev** [comment]
I will make the last comment as separate issue

**Comment by EvgenyKarataev** [comment]
For it worked to delete Apache Tomcat from Java Build Path from BLL, DAL, and Util project
